package/gluon-mesh-vpn-wireguard/src/Makefile

```diff
@@ -1,6 +1,9 @@
-all: respondd.so
+all: respondd.so key-translate
 
 CFLAGS += -Wall -Werror-implicit-function-declaration
 
+key-translate: key-translate.c
```

Comment:
perhaps a more specific name such as `gluon-mesh-vpn-key-translate`, `fastd-key-translate`, or smth else?

Reply:
@lemoer suggested `gluon-mesh-vpn-key-translate` a few days before,
I'd like to keep its title generic, as it might even be possible to reverse the pubkeytranslation.

Reply:
resolved in eee7377d034c8a622d2dbe165454c450aa53e140